### DISCUSSION TITLE
How to upgrade pgvector on supabase local dev docker

### DISCUSSION THREAD

**Discussion by jocubeit** [supabase/supabase#17930] [Questions] [answered]
Current version is 0.4.0, would like 0.5.0 to take advantage of HNSW indexing.

**Comment by jocubeit** [comment]
Is this planned, or is there a way I can do this myself?

**Reply by kiwicopple** [reply]
you just need to bump the version here @jocubeit : https://github.com/supabase/supabase/blob/072222523afe612d575c2ff7e73e514692f545af/docker/docker-compose.yml#L348

We pin the docker image versions then update them occasionally. We used to use "latest" but it was both insecure and it would cause problems if there was a broken image version.

I'll add docs/instructions for this in the main docs

**Comment by kiwicopple** [comment, marked as answer]
you just need to bump the version here @jocubeit :

[supabase/docker/docker-compose.yml](https://github.com/supabase/supabase/blob/072222523afe612d575c2ff7e73e514692f545af/docker/docker-compose.yml#L348)

Line 348 in [0722225](https://github.com/supabase/supabase/commit/072222523afe612d575c2ff7e73e514692f545af)
 image: supabase/postgres:15.1.0.117 

We pin the docker image versions then update them occasionally. We used to use "latest" but it was both insecure and it would cause problems if there was a broken image version.

I'll add docs/instructions for this in the main docs

**Reply by kiwicopple** [reply]
Added a pull request for the docs here: https://github.com/supabase/supabase/pull/18160

**Reply by jocubeit** [reply]
Awesome, thank you!
